README.md

```diff
@@ -52,6 +52,14 @@ Note that you can use one or the other but not both. Chore will quit and make fu
 
 ### Tips for configuring Chore
 
+For Rails, it can be necessary to add the directory you store your jobs in to the eager loading path,
+found in application.rb. You likely need a similar approach for most apps using jobs, unless you places
+them into a directory that is already eager loaded by default. One example of this might be:
+
+```ruby
+config.eager_load_paths += File.join(config.root, "app", "jobs")
```

Comment:
It is a rails thing, but it means if you follow this (and only this) Chore won't run correctly in dev mode. Historically we've had to actively load them in an initializer to ensure we have the classes loaded when we need them.